src/components/Navigation.tsx

```diff
@@ -20,11 +24,34 @@ const Navigation: FC = () => {
   const [projectName, setProjectName] = useState(
     project && !isLoading ? project.name : "default",
   );
+  const { hasCustomVolumeIso } = useSupportedFeatures();
+  const enablePermissions = enablePermissionsFeature();
```

Comment:
Shall we merge the enablePermissions into the useSupportedFeatures?

Reply:
I would prefer to keep useSupportedFeatures pure for lxd version and api extension related tasks wdyt?